<!-- -->
<launch>
  <arg name="OA_fis_file"   default="/home/one/catkin_ws/src/nnca_cluster/obstacle_avoidance.fis"/> 
  <arg name="GO_fis_file"   default="/home/one/catkin_ws/src/nnca_cluster/goal_oriented4.fis"/> 

  <node name="nnca_cluster_node" pkg="nnca_cluster" type="nnca_cluster_node"  output="screen" respawn="false">
    <param name="vel_factor"    type="double" value="1" />
    <param name="turn_factor"    type="double" value="1" />
    <param name="lock_angle"    type="double" value="75" />
    <param name="turning_angle"    type="double" value="30" />
    <param name="orbbec_turning_angle"    type="double" value="15" />
    <param name="mid_distance"    type="double" value="0.5" />
    <param name="left_distance"    type="double" value="0.3" />

    <remap from="/scan" to="/T4_scan"/>

  </node>


</launch>
